recipes/pylbm/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set version = "0.3.2" %}
+
+package:
+  name: pylbm
+  version: {{ version }}
+
+source:
+  url: https://github.com/pylbm/pylbm/archive/{{ version }}.tar.gz
+  sha256: 9a9263a5f14fbbd6a65b73c7e904c1add789f3334228f9535309d533a104b552
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [win]
```

Can this be `noarch`? It doesn't matter if Windows fails due to missing dependencies
```suggestion
  noarch: python
```
If not you need to remove the version constraint from Python and use `skip: true  # [win or py<34]` instead

---

this package can be `noarch` with `Python>=3.4`. I removed it due to the windows build failure.